plugins/stackstorm/src/components/ExecutionsTable/ExecutionPanel.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { CodeSnippet, Progress } from '@backstage/core-components';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
+import { Execution, stackStormApiRef } from '../../api';
+import useAsync from 'react-use/lib/useAsync';
+import { Alert } from '@material-ui/lab';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  Typography,
+  withStyles,
+} from '@material-ui/core';
+import { Status } from './Status';
+
+const useStyles = makeStyles(theme => ({
+  table: {
+    maxWidth: '50%',
+    flex: 'i',
+  },
+  title: {
+    paddingTop: theme.spacing(2),
+    fontSize: 14,
+    fontWeight: 'bold',
+    textTransform: 'uppercase',
+  },
+  card: {
+    borderBottom: `2px solid ${theme.palette.divider}`,
+  },
+}));
+
+const THead = withStyles(() => ({
+  root: {
+    paddingLeft: 0,
+  },
+}))(TableCell);
+
+const TRow = withStyles(theme => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: theme.palette.background.paper,
+    },
+  },
+}))(TableRow);
+
+const ExecutionCard = ({ e }: { e: Execution }) => {
+  const config = useApi(configApiRef);
+  const classes = useStyles();
+
+  const webUrl = `${config.getString('stackstorm.webUrl')}/?#/history/${e.id}`;
+
+  return (
+    <Card className={classes.card}>
+      <CardContent>
+        <Table className={classes.table} size="small">
```

Comment:
There's a Table in core-components - did you intentionally opt out of it or was that an oversight? It's a bit simplified for better or worse, but importantly it has consistent styling with everything else. https://backstage.io/storybook/?path=/story/data-display-table--default-table

Reply:
This was intentional. Desired output is not really a table but definition list. I found it easier to achieve desired result with stock material table.

![Screenshot 2023-02-16 at 10 30 44](https://user-images.githubusercontent.com/24315860/219340343-8e39a8c8-67bf-41de-8774-ac44eea6fea4.png)

fyi I'm not a FE engineer so if there is a better component to use to achieve desired result please let me know.

Reply:
Alright! Yeah sure, no problem then.